src/containers/Passphrase.jsx

```diff
@@ -21,13 +22,32 @@ const mapStateToProps = state => ({
   passphrase: state.passphrase
 })
 
-const showSuccessThenReload = t => {
-  Alerter.info(t('PassphraseView.reload'))
+export const getRedirectUrlsFromURLParams = urlParamsStr => {
```

Comment:
Nit: parse would be explicit about the need to parse the URL.

Reply:
See https://github.com/cozy/cozy-settings/pull/269/commits/819ef5ef41080821db2b288feda4b4599ea34e2d